packages/common-config/hardhat.config.ts

```diff
@@ -73,6 +71,7 @@ const config = {
     apiKey: {
       mainnet: process.env.ETHERSCAN_API_KEY,
       rinkeby: process.env.ETHERSCAN_API_KEY,
+      goerli: process.env.ETHERSCAN_API_KEY,
```

Comment:
We were missing the goerli config 😬

Reply:
not sure why that was missing but :pray: